index.js

```diff
@@ -101,30 +180,45 @@ module.exports = function(babel) {
         }
 
         // use "styled" as new default import, only if there's no such variable in use yet
-        const newName = path.scope.hasBinding("styled")
-          ? path.scope.generateUidIdentifier("styled").name
-          : "styled";
+        const createUniqueIdentifier = name =>
+          path.scope.hasBinding(name) ? path.scope.generateUidIdentifier(name) : t.identifier(name);
         let newImports = [];
-        let useDefaultImport = false;
+        let emotionImports = {};
```

Comment:
What does this do?

Reply:
Depending on what features are needed, we potentially need to import the `default` export or `css` or `cx` from `react-emotion`.
These imports are added to the code [here](https://github.com/TejasQ/babel-plugin-glamorous-to-emotion/blob/899ec9772c3fdc78c2892e327bbec0bcedbee9b5/index.js#L238-L245)